src/Logic.hs

```diff
@@ -205,13 +205,13 @@ ensureCloned config =
 
 data Event
   -- GitHub events
-  = PullRequestOpened PullRequestId Branch Sha Text Username -- PR, branch, sha, title, author.
+  = PullRequestOpened PullRequestId Branch BaseBranch Sha Text Username -- PR, branch, base branch, sha, title, author.
   -- The commit changed event may contain false positives: it may be received
   -- even if the commit did not really change. This is because GitHub just
   -- sends a "something changed" event along with the new state.
   | PullRequestCommitChanged PullRequestId Sha -- PR, new sha.
   | PullRequestClosed PullRequestId            -- PR.
-  | PullRequestEdited PullRequestId Text       -- PR, new title.
+  | PullRequestEdited PullRequestId Text BaseBranch -- PR, new title, new base branch.
```

Comment:
Another Haddock convention, if you change the comment style to `-- ^`, it attaches the comment to the constructor that came before, which is really useful if you hover over a constructor in your editor  to see what everything means.

Reply:
Thanks for the tip, but I think I'd need a new PR to fix all the comments. :sweat_smile: